module/TaskManagement/src/TaskManagement/Controller/TransitionsController.php

```diff
@@ -93,6 +118,31 @@ public function invoke($id, $data) {
 					$this->response->setStatusCode ( 403 );
 				}
 				break;
+			case "close":
+				//recupero tutti i task accettati per i quali è stato superato il limite per assegnare gli share
+				$tasksFound = $this->taskService->findAcceptedTasksBefore($this->getIntervalForCloseTasks());
```

Comment:
Arrivo in questo controller attraverso una rotta definita come `/task-management/tasks[/:id][/:controller]` . Immagino allora che devo definire una nuova rotta 
`/task-management/tasks[/:controller]` e definire in `TransitionsController` un metodo `create` . E' corretto?